instrumentation/koala/lib/opentelemetry/instrumentation/koala/patches/instrumentation.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Koala
+      module Patches
+        # Module to prepend to Koala::Facebook::API for instrumentation
+        module Instrumentation
+          VERBS_TO_SPAN_NAMES = Hash.new { |h, k| h[k] = "Koala #{k}" }
+
+          def graph_call(path, args = {}, verb = 'get', options = {}, &post_processing)
+            tracer.in_span(
+              VERBS_TO_SPAN_NAMES[req.method],
+              {
+                'koala.verb' => verb,
+                'koala.path' => path
+              },
+              kind: :client
+            ) do |span|
+              super(path, args, verb, options, &post_processing)
```

Comment:
Do we also want to:
```suggestion
              OpenTelemetry::Common::HTTP::ClientContext.with_attributes('peer.service' => 'facebook') do
                super(path, args, verb, options, &post_processing)
              end
```
or should that be the caller's responsibility?

If we opt to add ^ then we also need to add the dependency on `opentelemetry-common`.

Reply:
I'd definitely like to see this change come in.

Reply:
Great idea, didn't think of that!!